test/functional/options/defaults_spec.lua

```diff
@@ -380,3 +382,279 @@ describe('XDG-based defaults', function()
     end)
   end)
 end)
+
+
+describe('stdpath()', function()
+  -- Returns an OS friendly version of the unix_path.
+  local function alter_path(unix_path)
+    if helpers.iswin() then
+      local altered, _
+      altered, _ = unix_path:gsub('/', '\\')
+      return altered
+    else
+      return unix_path
+    end
+  end
```

Comment:
Now there is function in helpers which does the same thing: `alter_slashes` (it was not when PR was originally created).